test/parallel/test-timers-api-refs.js

```diff
@@ -1,20 +1,24 @@
 'use strict';
 const common = require('../common');
-const assert = require('assert');
+const timers = require('timers');
```

Comment:
is this possible? IIRC from before, you couldn't actually manually load internal modules from user code like tests.